src/themes/default/components/core/blocks/Header/Header.vue

```diff
@@ -3,23 +3,23 @@
     <header class="brdr-bottom bg-white brdr-c-alto"  :class="{ 'is-visible': navVisible }">
       <div class="container">
         <div class="row between-xs middle-xs px15" v-if="!isCheckout">
-          <div class="col-sm-4 col-xs-2 middle-xs">
+          <div class="col-xs-2 col-sm-2 middle-xs">
```

Comment:
@pakiam are you sure to declare the same column on each resolution ? I mean when you declared 2 columns on xs, you don't have to repeat it on sm if you don't need any change

Reply:
@mercs600 Well, I think it`s helps read code easily, plus you can count cols faster. If you want I can remove repeating. But when you watch at this you can 100% sure which columns disappear on -md- for example, and which stays on -sm- without thinking.
PS This is how Header looks like on master branch 
![image](https://user-images.githubusercontent.com/11723092/35349507-2c94db18-014c-11e8-8107-b0d0cdf08492.png)

Reply:
@pakiam I now what you mean, but in my opinion and from code perspective this is totally redundant. Grid frameworks are being builded mobile first, so when you see only col-xs-2 you should know at the same time you have 2 col on other resolution. Please remove repeating ;-)